pyseq.py

```diff
@@ -876,6 +1060,40 @@ def get_sequences(source):
             seq = Sequence([item])
             seqs.append(seq)
 
-    log.debug('time: %s' % (datetime.now() - start))
-
-    return list(seqs)
+    log.debug("time: %s" %(datetime.now() - start))
+    if not stereo:
+        log.info("added sequences: %s" %(seqs))
+        return seqs
+    else:
+        seqs.sort()
```

Comment:
There must be a better way to do this.  Something closer to 'diff', like: re.split on views, if split parts match between seqs, they're a pair?  Or am I out of stereo sequence practice?